.github/workflows/notebooks-digest-updater-downstream.yaml

```diff
@@ -0,0 +1,107 @@
+# The aim of this GitHub workflow is to update the params.env file with the latest digest.
+on:
+  workflow_dispatch:
+    inputs:
+      branch:
+        required: true
+        description: "Provide the name of the branch you want to update ex master, rhods-x.xx etc"
+      release-n:
+        required: true
+        description: "Provide release N version of the notebooks ex 2023a"
+
+jobs:
+  initialize:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+    steps:
+      - name: Install Skopeo CLI
+        shell: bash
+        run: |
+          sudo apt-get -y update
+          sudo apt-get -y install skopeo
+
+      # Checkout the release branch
+      - name: Checkout release branch
+        uses: actions/checkout@v3
+        with:
+          ref:  ${{ inputs.branch }}
+
+      # Create a new branch
+      - name: Create a new branch
+        run: |
+         git checkout -b digest-updater
```

Comment:
thinking to use a branch more specific e.g `notebooks-digest-updater` in case other image wants to update SHA1 in the same fashion